cli/requirements.txt

```diff
@@ -3,4 +3,5 @@ tox>=2.2, <3.0
 wheel>=0.24.0, <1.0
 pytest>=2.9.1
 teamcity-messages>=1.20
+PyInstaller==3.2.1
```

Comment:
It's not a "new" dependency so much as it is a consolidation of installing this dependency as part of the standard `make env` step. Previously, the scripts in the CI did:
```
echo "Building binary with pyinstaller"
cd cli
pip install -r requirements.txt
pip install -e .
pip install pyinstaller==3.1.1
pyinstaller binary/binary.spec
```

Which seems completely unnecessary to me. I guess the idea was that if you are only developing and don't need to build the binary, then you don't need this. I think it makes more sense to include it always though.